libnavigation-base/src/main/java/com/mapbox/navigation/base/options/RoutingTilesOptions.kt

```diff
@@ -1,5 +1,7 @@
 package com.mapbox.navigation.base.options
 
+import com.mapbox.common.TileStore
+import com.mapbox.navigation.base.internal.tilestore.TileStoreProvider
 import java.net.URI
 
```

Comment:
I think it's a good moment to also add some clarifications around `tileBaseUri` and how to disable from a client point of view _Offline_ and _Predictive Caching_ altogher. Capturing from @mskurydin 

> provide an empty host in the endpoint configuration - the navigator will work in the fallback mode all the time (route line following; no full map-matching; no map-matching in free drive)

> if users want to sacrifice user experience and just use active guidance with a route line only, the best thing you can do - provide an empty host name to nav-native

Reply:
done